tutorhastexo/patches/lms-env

```diff
@@ -1,4 +1 @@
-"ADDL_INSTALLED_APPS": ["hastexo"],
-"XBLOCK_SETTINGS": {
-    "hastexo": {{ HASTEXO_XBLOCK_SETTINGS|tojson(indent=4) }}
-}
+"ADDL_INSTALLED_APPS": ["hastexo"]
```

Comment:
Don't we want this in `openedx-common-settings` too, so that we _add_ `hastexo` to the `ADDL_INSTALLED_APPS` list rather than overriding the list altogether?

Reply:
I believe `ADDL_INSTALLED_APPS` is not defined anywhere, so we cannot add to it. But we can probably define it via `openedx-common-settings` instead of `lms-env`, yes.

Reply:
If you're modifying settings directly and not using `lms.env.json` at all, I think there isn't even any need to use `ADDL_INSTALLED_APPS`. 

Instead, you should be able to do:

```python
INSTALLED_APPS.append("hastexo")
```

That's [what the LMS itself does](https://github.com/openedx/edx-platform/blob/master/lms/envs/production.py#L313) (and [the CMS, too)](https://github.com/openedx/edx-platform/blob/master/cms/envs/production.py#L273).

What do you think?

Reply:
Ah, yes, makes sense, I'll update it :)